frontend/src/lib/ForwardMessageCache.test.ts

```diff
@@ -64,52 +60,6 @@ function createRefMsg(msg: ForwardMsg): ForwardMsg {
   })
 }
 
-/**
```

Comment:
We no longer mock HTTP calls in `ForwardMessageCache`; we just mock the return value of `StreamlitEndpoints.fetchCachedForwardMsg`